### DISCUSSION TITLE
Dockable Tool Windows

### DISCUSSION THREAD

**Discussion by Hanprogramer** [godotengine/godot-proposals#4744] [Editor]
I think it would be good to see something like in visual studio(not exactly like it) or vscode. This is one of the problems i had so far with godot editor, plus you cant really resize everything

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/28 and https://github.com/godotengine/godot-proposals/issues/4565.
